### PR TITLE
#160724661 populate readme

### PR DESCRIPTION
**Pull request by ddouglasz**
**What does this PR do?**

- This pull request populates the README.md file with details about the project.

**Description of Task to be completed?**
  - populate readme
  - refactor mongo config file to use dovent variable

**How should this be manually tested?**
- Clone the app to your local machine
- Navigate to README.md folder

**Any background context you want to provide?**
- None

[#160724661](https://www.pivotaltracker.com/story/show/160909357)

Screenshots (if appropriate):
- None
[Delivers #160724661]

### CODE REVIEW

File: server/config/mongoConfig.js [hunk 1]
```diff
@@ -1,12 +1,16 @@
...
-const mongoDB = 'mongodb://127.0.0.1/igive';
+// const mongoDB = 'mongodb://127.0.0.1/igive';
+dotenv.config();
+const mongoDB = process.env.MONGO_URL;
 mongoose.connect(mongoDB);
 // Get Mongoose to use the global promise library
 mongoose.Promise = global.Promise;
 // Get the default connection
 const db = mongoose.connection;
+console.log('we are in');
```
Comment (Rhotimee) [inline server/config/mongoConfig.js]: Hey @ddouglasz, do you think this console.log is necessary here? 
If it is, let's log a more descriptive message like `Mongo connected`
else, let's remove it.

File: server/config/mongoConfig.js [hunk 2]
```diff
@@ -1,12 +1,16 @@
 import mongoose from 'mongoose';
+import dotenv from 'dotenv';
 
 // Set up default mongoose connection
-const mongoDB = 'mongodb://127.0.0.1/igive';
+// const mongoDB = 'mongodb://127.0.0.1/igive';
```
Comment (Rhotimee) [inline server/config/mongoConfig.js]: I feel we should remove commented code.

Comment (ddouglasz) [reply server/config/mongoConfig.js]: Okay great. On it. Thank you.

File: server/config/mongoConfig.js (hunk 1 above)
Comment (ddouglasz) [reply server/config/mongoConfig.js]: Okay on it. Thanks

File: server/config/mongoConfig.js (hunk 2 above)
Comment (ddouglasz) [reply server/config/mongoConfig.js]: Done.

File: server/config/mongoConfig.js (hunk 1 above)
Comment (Rhotimee) [reply server/config/mongoConfig.js]: 👍🏽

**Pull request merged**